get-involved/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+hero-image: url(../assets/images/stock-images/pie-staff.jpg)
+hero-message: Get Involved!
+---
+
+<!-- WHAT WE DO -->
+<div class="container">
+    <h1 style="text-align: center">Join PiE Staff</h1>
+    <!--<h2>WHAT WE DO</h2>-->
+</div>
+<div class="rc-row vertical-align" >
+    <div class="text-col text-left" id="fall-row" >
+        <div class="graphic-description">
+            <h3>ENGINEERING</h3>
+            <p>We're mechanical engineers, software engineers, electrical engineers. We build, design, reiterate, and
```

Comment:
I like this, but wondering if it's possible to also include something about the business, event planning, etc. side of PiE?